src/app/components/common/WhiteWrapper/WhiteWrapper.tsx

```diff
@@ -82,7 +83,7 @@ const WhiteWrapper = (props: Props) => {
       action: 'click',
       label: 'operator',
     });
-    window.open(`${config.links.EXPLORER_URL}/operators/${explorerLink}/?version=${config.links.EXPLORER_VERSION}&network=${config.links.EXPLORER_NETWORK}`, '_blank');
+    window.open(`${config.links.EXPLORER_URL}/operators/${explorerLink}/?version=${config.links.EXPLORER_VERSION}&network=${ENV().NETWORK}`, '_blank');
```

Comment:
The same, just go over all such places and fix

Reply:
done